packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -318,6 +437,24 @@ const EDiscovery = WebexPlugin.extend({
     }
   },
 
+  /**
+   * Retrieves a uuid string from a report url
+   * @param {String} reportUrl - full destination address (including report id parameter) for the http request being sent
+   * e.g. 'http://ediscovery-intb.wbx2.com/ediscovery/api/v1/reports/3b10e625-2bd5-4efa-b866-58d6c93c505c'
+   * @returns {String} - uuid of the report
+   */
+  _getReportIdFromUrl(reportUrl) {
+    if (reportUrl) {
+      const uuids = reportUrl.match(/[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}/);
+
+      if (uuids && uuids.length > 0) {
+        return uuids[0];
+      }
+    }
+
+    return '';
+  },
+
```

Comment:
New function used by the three new APIs to allow them to extract the report id from the url parameter